assets/src/edit-story/components/keyboardShortcutsMenu/keyboardShortcutsMenuContext/provider.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useState, useMemo, useCallback } from 'react';
+import { trackEvent } from '@web-stories-wp/tracking';
+
+/**
+ * Internal dependencies
+ */
+import Context from './context';
+
+const KeyboardShortcutsMenuProvider = ({ children }) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggle = useCallback(
+    (e) => {
+      e.preventDefault();
+      trackEvent('shortcuts_menu_toggled', {
```

Comment:
not new, just lifting from previous toggle since we couldn't reach that at the secondaryMenu level where we're tracking state for the 3 popups.

Reply:
oh beautiful. ignore my question above